cms/static/js/views/modals/edit_xblock.js

```diff
@@ -9,10 +9,10 @@ define(["jquery", "underscore", "gettext", "js/views/modals/base_modal", "common
         "strict mode";
 
         var EditXBlockModal = BaseModal.extend({
-            events : {
+            events: _.extend({}, BaseModal.prototype.events, {
```

Comment:
It's really frustrating that this has to be done everywhere. Is this a known change in the newer version of Backbone?